notebooks/osmFISH.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# coding: utf-8
+#
+# EPY: stripped_notebook: {"metadata": {"kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.6.5"}}, "nbformat": 4, "nbformat_minor": 2}
+
+# EPY: START markdown
+# ## Loading the data into Starfish
+# EPY: END markdown
+
+# EPY: START code
+from starfish.io import Stack
```

Comment:
Perhaps outside of scope of this PR, but I think we should stop using Stack in favor of ImageStack.

Reply:
Interesting. We'd probably have to use `ImageStack.from_url()` and point it at the _hybridization_.json instead of the experiment.json. @ttung Would that work? 

@dganguli If we did it this way, it wouldn't read in the `aux_images`. Is that desirable?

Reply:
I think we should rename Stack → Experiment